TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Extensions/NpgsqlExtensions.cs

```diff
@@ -0,0 +1,117 @@
+using System.Text.Json;
+using Npgsql;
+using NpgsqlTypes;
+
+namespace TeachingRecordSystem.Core.DataStore.Postgres.Extensions;
```

Comment:
Can we move this up a level (saves an extra `using` to have the methods visible)?